api/src/config/githubConfig.ts

```diff
@@ -0,0 +1,36 @@
+import { readFileSync } from 'fs'
+import { logger } from '@bcgov/common-nodejs-utils';
+import path from 'path'
+
+const getBasePathWithFile = (file) => {
+  let filePath
+  
+  if (
+    (process.env.NODE_ENV === 'production'|| process.env.NODE_ENV === 'development') &&
```

Comment:
What benefit does the `(process.env.NODE_ENV === 'production'|| process.env.NODE_ENV === 'development')` add as we only have two NODE_ENV environments which are both production and development.

Reply:
Oh, it just useful when I do local development. will remove

Reply:
Wasn't a critism, if this is a valuable add, I more meant, you could remove the entire line as it will always be True given there are only two NODE ENVS.